example/client-server/client.js

```diff
@@ -16,10 +20,10 @@ CartItem.attachTo(remote);
 
 // call the remote method
 CartItem.sum(1, function(err, total) {
-  console.log('result:', err || total);
+  g.log('result:', err || total);
```

Comment:
This is probably slightly better:
`console.log(g.f('result: %s', err || total));`